synedrion/src/uint/secret_signed.rs

```diff
@@ -0,0 +1,710 @@
+use core::ops::{Add, Mul, Neg, Sub};
+
+use crypto_bigint::{
+    rand_core::CryptoRngCore,
+    subtle::{Choice, ConditionallySelectable, ConstantTimeLess, CtOption},
+    zeroize::Zeroize,
+    BitOps, Bounded, CheckedAdd, CheckedMul, CheckedSub, Integer, NonZero, RandomMod, ShlVartime, WrappingAdd,
+    WrappingMul, WrappingNeg, WrappingSub,
+};
+
+use super::{HasWide, PublicSigned, SecretUnsigned};
+use crate::tools::Secret;
+
+/// A wrapper over secret unsigned integers that treats two's complement numbers as negative.
+#[derive(Debug, Clone)]
+pub(crate) struct SecretSigned<T: Zeroize> {
+    /// bound on the bit size of the absolute value (that is, `abs(value) < 2^bound`)
```

Comment:
```suggestion
    // bound on the bit size of the absolute value (that is, `abs(value) < 2^bound`)
```

(or move this info to the docs)

Reply:
Not sure what you mean, isn't it in the docs already? It's a doc comment

Reply:
You are correct, my bad. Maybe capitalize to "Bound" though?